crates/nu-cli/src/commands/sort_by.rs

```diff
@@ -65,6 +65,18 @@ fn sort_by(args: CommandArgs, registry: &CommandRegistry) -> Result<OutputStream
         let (SortByArgs { rest }, mut input) = args.process(&registry).await?;
         let mut vec = input.drain_vec().await;
 
+        for sort_arg in rest.iter() {
+            let mut match_test = get_data_by_key(&vec[0], sort_arg.borrow_spanned());
+            if match_test == None {
+                yield Err(ShellError::labeled_error(
+                    "Invalid column",
+                    "Can not find column to sort by",
```

Comment:
Two things here:

First, you should swap the first and second argument. The first argument is the more descriptive version, and the second is the one that labels the part of the pipeline that failed (so it should be short).

Also, to stay consistent with our messaging elsewhere, no capitalization on the first word for the label (the second argument) 🙂

Reply:
Thank you, I have addressed these comments!